lib/updatehub/updatehub_timer.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_MODULE_NAME updatehub_timer
+#define LOG_LEVEL CONFIG_UPDATEHUB_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
We could utilize the same logging module here like this
```
#include <logging/log.h>
LOG_MODULE_DECLARE(updatehub, CONFIG_UPDATEHUB_LOG_LEVEL);
```

Reply:
I'll send update.